### PR TITLE
DDH Reflow - Join a Community

### PR DESCRIPTION
**Pull request by alohaas**
@maria @abeyang @russell @john 
[UX Discussion](https://app.asana.com/0/29962218663915/68407110736063)
[DDH Changes Task](https://app.asana.com/0/66797129970807/66797129970817)
[Mockup](https://projects.invisionapp.com/share/59509LDMY#/screens)

Desktop
screen shot 2015-12-04 at 1 54 31 pm

Mobile
screen shot 2015-12-04 at 1 55 09 pm

### Notes:
- The number of Slack members "online" is actually the total number of members on the DDH slack. Using the Slack API to get the members online requires a paid Slack account. Should this say "# Members" rather than "# Online"?
- The number of Twitter followers is a static number right now, which is reflective of the @duckduckhack account. I'm working on using the Twitter spice to get that number synced up to the Twitter API.
- Meetup number will need to be updated manually every now and then.
- Currently the entire community "card" is a link. It has no hover state outside of the cursor change. I considered having the card slide up a little on hover, but wanted @abeyang to weigh in on that.

### CODE REVIEW

**Comment by russellholt** [comment]
:+1:

File: share/site/duckduckhack/index_content.tx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="blk  blk--hero  blk--hero--alt  js-hero-slides"><a class="anchor ddh-anchor js-anchor" name="top"></a>
+ <div class="blk  blk--hero  blk--hero--alt  js-hero-slides"><a class="anchor ddh-anchor js-anchor" name="top"></a>
```
Comment (bsstoner) [inline share/site/duckduckhack/index_content.tx]: any reason for this extra space?

Comment (alohaas) [inline share/site/duckduckhack/index_content.tx]: Nope. Should be fixed.

Comment (bsstoner) [inline share/site/duckduckhack/index_content.tx]: :+1:

File: lib/DDG/Publisher/Site/Duckduckhack/Root.pm [hunk 2]
```diff
@@ -17,9 +17,10 @@ sub pages {
 		no_search_header => 1,
 		hero_header => 1,
 		no_hero_header_icon => 1,
-		hero_header_link => 'https://duck.co/duckduckhack/ddh-intro',
+		hero_header_link => '',
 		hero_header_link_text => "Docs",
 		hero_alt => 1,
+        no_footer_arrow => 1,
```
Comment (bsstoner) [inline lib/DDG/Publisher/Site/Duckduckhack/Root.pm]: can you fix the indentation here please? :)

Comment (alohaas) [inline lib/DDG/Publisher/Site/Duckduckhack/Root.pm]: Sure thing!

**Pull request merged**